backend/database.py

```diff
@@ -1,26 +1,226 @@
+        u_role varchar(1) not null
+    );
+    '''
+    c.execute(command)
+
+    command = '''
+    CREATE TABLE Classes (
+        c_classkey int primary key not null,
+        c_classname varchar(255) DEFAULT "TBA" not null,
+        c_teacher varchar(255) DEFAULT "TBA" not null,
+        c_time varchar(255) DEFAULT "TBA" not null,
+        c_seats int not null
+    );   
+    '''
+    c.execute(command)
+
+    command = '''
+    CREATE TABLE Students_in_Classes (
+        sc_classkey int not null,
+        sc_name varchar(255) not null,
+        sc_grade int not null,
+        primary key (sc_classkey, sc_name)
+    )
+    '''
+    c.execute(command)
+    c.commit()
+
+def insert_user(name, password, type, c):
+    command = '''
+    INSERT INTO Users
+    values('''+\
+    "\""+str(name)+"\", "+\
+    "\""+str(password)+"\", "+\
+    "\""+str(type)+"\");"
+    
+    c.execute(command)
+    c.commit()
+
+def delete_user(name, c):
+    get_result = get_user(name, c)
+
+    if get_result != None:
+        user_type = get_result[2]
+        command = "DELETE FROM Users WHERE u_name = \""+name+"\";"
+        c.execute(command)
+        c.commit()
+        if user_type == "s":
+            command = "DELETE FROM Students_in_Classes WHERE sc_name = \""+name+"\";"
+            c.execute(command)
+            c.commit()
+        
+        elif user_type == "t":
+            command = "UPDATE Classes SET c_teacher = \"TBA\" WHERE c_teacher = \""+name+"\";"
+            c.execute(command)
+            c.commit()
+    else:
+        print("ERROR: User not found\n")
+    
+    
+def student_add_class(name, classkey, c):
+    command = '''
+    INSERT INTO Students_in_Classes
+    values('''+str(classkey)+\
+    ", \""+str(name)+"\", 100);"
+    c.execute(command)
+    c.commit()
+
+def student_drop_class(name, classkey, c):
+    command = '''
+    DELETE FROM Students_in_Classes '''+\
+    "WHERE sc_name = \""+str(name)+"\"and sc_classkey = "+str(classkey)+";"
+    c.execute(command)
+    c.commit()
+
+def insert_class(key, name, teacher, time, seats, c):
+    command = '''
+    INSERT INTO Classes
+    values('''+str(key)+", "+\
+    "\""+str(name)+"\", "+\
+    "\""+str(teacher)+"\", "+\
+    "\""+str(time)+"\", "+str(seats)+");"
+    
+    c.execute(command)
+    c.commit()
+
+def student_grades_in_student_class(name, c):
+    command = '''
+    SELECT c_classname, sc_grade
+    FROM Students_in_Classes, Classes '''+\
+    "WHERE sc_name = \""+str(name)+"\" and sc_classkey = c_classkey"
+    cur = c.cursor()
+    cur.execute(command)
+    result = cur.fetchall()
+    print(result)
+    return result
+
+def edit_student_grade(student_name, classkey, grade, c):
+    command = "UPDATE Students_in_Classes "+\
+    "SET sc_grade = "+str(grade)+" "+\
+    "WHERE sc_name = \""+str(student_name)+"\" and sc_classkey = "+str(classkey)+";"
+    c.execute(command)
+    c.commit()
+    
+
+""" ''if __name__ == '__main__':
```

Comment:
I'm kind of curious about this part. Why is there a giant string here?

Reply:
this is what happened when I use block comment on vscode python.
I'm not sure why it's not a bunch of #'s

Reply:
Weird.